src/promptflow/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -369,27 +369,26 @@ def _export_flow_connections(
         *,
         output_dir: Path,
     ):
-        from promptflow.batch._csharp_executor_proxy import CSharpExecutorProxy
-        from promptflow.contracts.flow import Flow as ExecutableFlow
-
-        executable = ExecutableFlow.from_yaml(
-            flow_file=Path(flow_dag_path.name), working_dir=flow_dag_path.parent.absolute()
-        )
-
-        with _change_working_dir(flow_dag_path.parent):
-            if executable.program_language == FlowLanguage.CSharp:
-                connection_names = SubmitterHelper.resolve_connection_names_from_tool_meta(
-                    tools_meta=CSharpExecutorProxy.generate_tool_metadata(
-                        working_dir=flow_dag_path.parent.absolute(),
-                    ),
-                    flow_dag=executable.serialize(),
-                )
+        flow: ProtectedFlow = load_flow(flow_dag_path)
+        with _change_working_dir(flow.code):
```

Comment:
flow with symlink and flow with additional includes may blame this.
flow.code may not point to correct path of flow.

Reply:
same with other places

Reply:
updated in comment. this function is used in `build`. For a built flow, there will be no additional includes, symlinks, variants, and such things.

Reply:
@crazygao  for code in `TestSubmitter`, similarly, things like variants, additional includes and so on will be resolved before resolving connections and I think this is expected.